Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -282,4 +289,30 @@ const std::string GameListItem::GetWiiFSPath() const
 
 	return ret;
 }
-
+bool GameListItem::isElfDol(const GameListItem& game)
```

Comment:
No, only the GameListItem one, since the string one is still used [here](#diff-2ebf3ad42e590cbc246f0a1f3e3e479aR151)